.circleci/config.yml

```
@@ -179,47 +179,199 @@ commands:
           paths:
             - << parameters.install_path >>
 
-defaults:
+  # --------------------------------------------------------------------------
+  # Build Commands
+
+  setup_prerelease_commit_hash:
+    steps:
+      - run:
+          name: Store commit hash and prerelease
+          command: |
+            if [[ $CIRCLE_BRANCH == release || -n $CIRCLE_TAG ]]; then
+              echo -n > prerelease.txt;
+            else
+              date -u +"nightly.%Y.%-m.%-d" > prerelease.txt;
+            fi
+            echo -n "$CIRCLE_SHA1" > commit_hash.txt
+
+  run_build:
+    steps:
+      - run:
+          name: Build
+          command: scripts/ci/build.sh
+
+  run_build_ossfuzz:
+    steps:
+      - run:
+          name: Build_ossfuzz
+          command: scripts/ci/build_ossfuzz.sh
+
+  run_proofs:
+    steps:
+      - run:
+          name: Correctness proofs for optimization rules
+          command: scripts/run_proofs.sh
+
+  run_soltest:
+    steps:
+      - run:
+          name: soltest
+          no_output_timeout: 30m
+          command: ./.circleci/soltest.sh
+
+  run_soltest_all:
+    steps:
+      - run:
+          name: soltest_all
+          no_output_timeout: 30m
+          command: ./.circleci/soltest_all.sh
+
+  run_cmdline_tests:
+    steps:
+      - run:
+          name: command line tests
+          no_output_timeout: 30m
+          command: .circleci/parallel_cli_tests.py
```

I'm fine with the PR, but just notice that some `command` use the current directory prefix (`./`) and others don't. Maybe we could also adjust the path of all commands?
```suggestion
          command: ./.circleci/parallel_cli_tests.py
```

---

Done. Removed the redundant `./` from all commands.